integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -985,69 +928,48 @@ private void createSecret() throws Exception {
         String.format(
             "kubectl label secret %s weblogic.domainUID=%s -n %s",
             secret.getSecretName(), domainUid, domainNS);
-    ExecResult result = ExecCommand.exec(labelCmd);
-    if (result.exitValue() != 0) {
-      throw new RuntimeException(
-          "FAILURE: command to label secret \""
-              + labelCmd
-              + "\" failed, returned "
-              + result.stdout()
-              + "\n"
-              + result.stderr());
-    }
+    TestUtils.exec(labelCmd);
   }
 
+  /**
+   * Generate domain values yaml using the doamin map
+   *
+   * @throws Exception
+   */
   private void generateInputYaml() throws Exception {
     Path parentDir =
         Files.createDirectories(Paths.get(userProjectsDir + "/weblogic-domains/" + domainUid));
     generatedInputYamlFile = parentDir + "/weblogic-domain-values.yaml";
     TestUtils.createInputFile(domainMap, generatedInputYamlFile);
   }
 
+  /**
```

Comment:
this doc appears to be missing the first sentence that explains what the method does.

Reply:
Done